drivers/builtin/include/mbedtls/pkcs12.h

```diff
@@ -1 +0,0 @@
-#include "private/pkcs12.h"
```

Comment:
The commit message says "Removed the files not present in the current development". How did these files end up here if they are not present on development?